numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -4232,3 +4232,36 @@ def add_newdoc(place, name, doc):
     array([ 0, 20, 20, 60, 20, 20])
 
     """)
+
+add_newdoc('numpy.core.umath', 'bit_count',
+    """
+    Computes the number of 1-bits in ``x``.
+    Analogous to the builtin `int.bit_count` or `popcount` in other languages.
```

Comment:
```suggestion
    Analogous to the builtin `int.bit_count` or ``popcount`` in other languages.
```
Since popcount doesn't link to anything.
Also, a single concrete example of one of these "other languages" would be nice.

Reply:
I have given C++ as an example as it has one in `std`, hope that's a good example.